tmail-backend/jmap/extensions-opensearch/pom.xml

```diff
@@ -14,11 +14,13 @@
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>jmap-extensions</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
we can add this dependency in the parent pom, 
then we don't need declare `version` here, and other pom